submissions/adekemiAwomolo/PhpAndDtabase/customersView.php

```diff
@@ -0,0 +1,57 @@
+<?php
+require_once('connection.php');
+    try{
+    global $pdo;
+    $sql = "SELECT single_orders.unit_price,
```

Comment:
Now your query is correct. But, you are putting this within a `try` block, which means this query will always run even when the cookie is set. I expected you to do this like your previous task. You put it within a function and only call the function when the cookie expires